app.js

```diff
@@ -1 +1,43 @@
-console.log("js file is connected")
\ No newline at end of file
+const rootUrl = "https://api.edamam.com/search";
+const recipeUrlElement = document.querySelector("#recipe-url");
+const labelElement = document.querySelector("#label");
+const imageElement = document.querySelector("#cake-img");
+let ingredientsListElement = document.querySelector(".ingredients");
```

Comment:
You don't need to define all this variables as a global ones. It would be better to define them just before you are going to use them inside the `fetchRecipies` function.
Or just go without defining a separate variables at all like this (I mean the correcponding lines inside `fetchRecipies` function):
```javascript
document.querySelector("#recipe-url").setAttribute("href", data.hits[0].recipe.url);
document.querySelector("#label").innerHTML = data.hits[0].recipe.label;
document.querySelector("#cake-img").src = data.hits[0].recipe.image;
```

Reply:
changed